docs/source/profiler.rst

```diff
@@ -464,12 +464,14 @@ Below is an breakdown of all the options.
       
       * is_case_sensitive - (Boolean)
       * stop_words - (List of Strings)
+      * top_k_chars - (Int)
+      * top_k_words - (Int)
```

Comment:
can you explain this specifically applies to the output of the counts?

Reply:
added